### PR TITLE
Bring POM / names in line with booster rules

### PR DESCRIPTION
**Pull request by lincolnthree**

### CODE REVIEW

**Review by geoand — APPROVED** [review]
LGTM but it's of course difficult to tell without actually testing :)

File: pom.xml [hunk 1]
```diff
@@ -3,29 +3,29 @@
...
 	</parent>
 
+	<groupId>io.openshift.booster</groupId>
+	<artifactId>spring-boot-istio-routing-parent</artifactId>
+	<version>1.5.12-1-SNAPSHOT</version>
+	<packaging>pom</packaging>
+
 	<properties>
 		<java.version>1.8</java.version>
 		<fabric8-maven-plugin.version>3.5.34</fabric8-maven-plugin.version>
```
Comment (metacosm) [inline pom.xml]: This should be removed along with the 2 next lines if I'm not mistaken.

**Review by metacosm — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (geoand) [reply pom.xml]: Unless Github is showing the wrong lines, why would the `artifactId`, `version` and `packaging` tags need to be removed?

Comment (metacosm) [reply pom.xml]: The comment applies to line 19-21 (and shows properly in here). It's just that the comment doesn't apply to a change so it's not highlighted :)

Comment (geoand) [reply pom.xml]: Thanks! That makes it clear :)

**Comment by lincolnthree** [comment]
Please review.

**Review by metacosm — APPROVED** [review]
LGTM (albeit without testing).

**Pull request closed without merging**